README.md

```diff
@@ -270,6 +272,20 @@ export SPHERE_CLIENT_SECRET="your-client-secret"
 #### ElasticIo
 _(Coming soon)_
 
+#### UserAgent
+A synchronous module that builds _user\_agent_ according to the standard specified [here](https://sphere.atlassian.net/wiki/x/7ACjB)
+
+sphere-node-sdk module must be installed in the node_modules because it's required to build the user_agent
```

Comment:
I don't quite understand what you mean.

Any project that wants to use this, will obviously have sdk in it's package.

I was just stating the obvious, or maybe I got you wrong.

If you can expantiate